Initial working elastic logging for sanity performance tests:

dkosteck opened the pull request:
An initial commit for the pull of results, tags, and timestamp to elastic for further processing/creating dashboards. This commit represents the beginning of being able to use elastic, not a comprehensive implementation. For this reason if other extensions (such as supporting CA Certs, more fields, etc.) are implemented they will follow in subsequent commits.

For information on the elastic and kibana deployment used for testing, see [this document](https://docs.google.com/document/d/1gOjT4hoc8jhe_G0qhyHFvCeKqrdiWQDGA4RKYCgaFps/edit?usp=sharing)

jianzzha reviewed (commented):
It might be a good idea to add a section in the readme file to explain this optional store capability/dev state.

dkosteck commented:
> It might be a good idea to add a section in the readme file to explain this optional store capability/dev state.

@jianzzha Thanks for pointing this out, I thought I had added a section to the README but apparently it got lost somewhere along the way. It's now updated
